dags/vm_resource.py

```diff
@@ -98,11 +98,14 @@ class ClusterName(enum.Enum):
   V4_32_CLUSTER = "mas-v4-32"
   V5E_4_CLUSTER = "mas-v5e-4"
   V5E_16_CLUSTER = "mas-v5e-16"
+  V4_8_MULTISLICE_CLUSTER = "v4-8-maxtext"
   V4_128_MULTISLICE_CLUSTER = "v4-bodaborg"
   V5E_16_MULTISLICE_CLUSTER = "v5e-16-bodaborg"
   V5E_256_MULTISLICE_CLUSTER = "v5e-256-bodaborg"
 
 
 class DockerImage(enum.Enum):
   XPK_JAX_TEST = "gcr.io/cloud-ml-auto-solutions/xpk_jax_test:latest"
+  MAXTEXT_JAX_STABLE = "gcr.io/tpu-prod-env-multipod/maxtext_jax_stable:latest"
+  MAXTEXT_JAX_NIGHTLY = "gcr.io/tpu-prod-env-multipod/maxtext_jax_nightly:latest"
```

Comment:
xpk won't pull the latest image if it's already on the node, so we still risk running stale images here. Since we'll be using the same tag `latest`, could you add a step to cache-image [here](https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/xlml/utils/xpk.py#L68)? cc @Obliviour for semantics of cache-image.

Alternatively, we could tag the images with the build date and use the new tag for each run.

Reply:
Ah interesting, glad you thought of this. Currently the maxtext images are also tagged by date, we would just need be careful of timing - e.g. the date requested has already been pushed. For this reason I like using `latest` instead of date as long as we could guarantee not stale, but using date also seems fine

Reply:
I think there may be a limit to how many images can be cached cc @Obliviour to confirm. However, if there is not a limit then both methods sgtm.

Reply:
I may be misunderstanding here but would removing all docker images here on the test host like we do in [maxtext](https://github.com/google/maxtext/blob/2e7e96744826665680a41e81cd393cf2d58cada0/.github/workflows/UnitTests.yml#L63-L66) guarantee us to pull the latest gcr copy?

Reply:
I changed it to use the date tagged image for now, since we know for sure it will pull the fresh image.